### PR TITLE
Update minimist to 1.2.4

### PR DESCRIPTION
**Pull request by wi-ry**
See https://www.npmjs.com/advisories/1179

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
   "dependencies": {
     "chalk": "1.1.3",
     "inquirer": "3.0.6",
-    "minimist": "1.2.0",
+    "minimist": "1.2.4",
```
Comment (mayanastasya) [inline package.json]: Hi, can we use `^1.2.0` or `~1.2.0` instead of pinning it down to a fixed version?

Comment (wi-ry) [reply package.json]: Good suggestion

**Comment by rajgoraya-asurion** [comment]
Please merge this. The current version of minimist@1.2.0 is flagged security issue by Synk  

✗ Prototype Pollution [https://snyk.io/vuln/SNYK-JS-MINIMIST-559764] in minimist@1.2.0
    introduced by opencollective@1.0.3 > minimist@1.2.0
  This issue was fixed in versions: 0.2.1, 1.2

**Review by rajgoraya-asurion — APPROVED** [review]

**Comment by znarf** [comment]
This package will not receive further update, we suggest to switch to it's replacement, [opencollective-postinstall](https://github.com/opencollective/opencollective-postinstall)  instead, this one doesn't have any dependency.

You can also consider removing the `postinstall` altogether, see: https://blog.opencollective.com/beyond-post-install/

**Comment by wi-ry** [comment]
Abandoning this PR based on @znarf's comments:
> This package will not receive further update, we suggest to switch to it's replacement, [opencollective-postinstall](https://github.com/opencollective/opencollective-postinstall) instead, this one doesn't have any dependency.
> 
> You can also consider removing the `postinstall` altogether, see: https://blog.opencollective.com/beyond-post-install/

**Pull request closed without merging**